installing/installing_bare_metal/bare-metal-using-bare-metal-as-a-service.adoc

```diff
@@ -0,0 +1,28 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="bare-metal-using-bare-metal-as-a-service"]
+= Using bare metal as a service
+include::_attributes/common-attributes.adoc[]
+:context: bare-metal-using-bmaas
+
+toc::[]
+
+{product-title}'s Bare Metal as a Service (BMaaS) feature enables you to provision and manage bare-metal nodes using the Metal^3^ API. These nodes, external to the {product-title} cluster, can host workloads requiring direct hardware access, such as legacy applications or high-performance computing tasks. BMaaS provides the following capabilities:
+
+* Provisioning of bare-metal nodes, including configuration and preparation for workloads.
+* Lifecycle management, such as power management, firmware updates, and decommissioning.
+
+As standalone systems, these nodes operate independently of the {product-title} cluster and support diverse workloads by integrating bare-metal resources with containerized and virtualized applications. BMaaS supports {op-system-base-full}, {op-system-first}, and Microsoft Windows on the bare-metal nodes.
```

Comment:
Some clarificataions on the OS side of things:
We DO NOT support RHCOS (RHEL CoreOS) outside OpenShift masters/workers. IIUC We are happy to support Fedora CoreOS. We do support RHEL. I think there is a plan to support Windows but we haven't tested it (certainly I have not). How about CentOS/Fedora? I will clarify (likely in a slack thread I will copy you on)
Existing, RHCOS-relevant thread: https://redhat-internal.slack.com/archives/C083P2FENTB/p1739790580460019

Reply:
also let's stress the importance of using the correct image type:

NOTE: Please ensure that the image used is appropriate for the server architecture and boot mode and is a full disk image (not a partition image). If in doubt, virt-filesystems tool from guestfs-tools package can be used to verify the contents of the image, e.g:

$ virt-filesystems --add image.qcow2 --filesystems --long
Name       Type        VFS   Label  Size        Parent
/dev/sda2  filesystem  vfat  -      209469440   -
/dev/sda3  filesystem  xfs   boot   1006632960  -
/dev/sda4  filesystem  xfs   root   9383706624  -
$

Reply:
I reduced it to just RHEL, but the text implies they can add others. We can add other specific examples if you want.